### DISCUSSION TITLE
`YAML.parse` not accepting options

### DISCUSSION THREAD

**Discussion by plummer-flex** [eemeli/yaml#214] [Q&A] [answered]
```
YAML.parse(fs.readFileSync(configPath, "utf-8"), null, {
    maxAliasCount: -1,
  });
```

returns `ReferenceError: Excessive alias count indicates a resource exhaustion attack`

**Comment by eemeli** [comment]
Could you provide the contents of the `configPath` file with which you're encountering this issue? Also, which version of `yaml` are you seeing this with?

**Comment by eemeli** [comment, marked as answer]
I think you're probably using `2.0.0-0`, and that this is fixed in `2.0.0-1`.

**Reply by plummer-flex** [reply]
> I think you're probably using `2.0.0-0`, and that this is fixed in `2.0.0-1`.

~Is the NPM package no longer updated?  https://www.npmjs.com/package/yaml~

nm, I see the readme in the repo
